src/Admin/Polyfills.php

```diff
@@ -32,10 +34,24 @@ public static function get_registration_action() {
 		return 'amp_register_polyfills';
 	}
 
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed() {
+		return ! Services::get( 'dependency_support' )->has_support();
+	}
+
 	/**
 	 * Runs on instantiation.
 	 */
 	public function register() {
+		$screen = get_current_screen();
+		if ( ! empty( $screen->is_block_editor ) ) {
+			return;
+		}
```

Comment:
This will prevent a fatal error from happening when polyfills are registered on the frontend:

```suggestion
		if ( function_exists( 'get_current_screen' ) ) {
			$screen = get_current_screen();
			if ( ! empty( $screen->is_block_editor ) ) {
				return;
			}
		}
```